crates/rustfinity-runner/Dockerfile

```diff
@@ -30,8 +33,15 @@ RUN cargo add \
 WORKDIR /app
 RUN cargo build
 
+ENV CARGO_HOME=/home/myuser/.cargo
+RUN mkdir -p $CARGO_HOME
 COPY --from=builder /app/target/release/rustfinity-runner /app/
 
+RUN useradd -u 1000 -ms /bin/bash myuser
+RUN chown -R myuser:myuser /home/myuser
```

Comment:
I need to change the owner here because, until line 43, all this is executed by root, so root is the owner.